cms/djangoapps/contentstore/views/tabs.py

```diff
@@ -72,6 +71,7 @@ def tabs_handler(request, course_key_string):
             'context_course': course_item,
             'tabs_to_render': tabs_to_render,
             'lms_link': get_lms_link_for_item(course_item.location),
+            'mfe_link': get_pages_and_resources_url(course_item),
```

Comment:
Can we use a more descriptive name here?
Given the proliferation of MFEs across the platform, it would be helpful to be more explicit about _which_ MFE we're linking to.